modules/curse/curse.gd

```diff
@@ -1,45 +1,58 @@
 extends Node2D
 
 #const var
-const target_angle_epsilon = 0.1* TAU
-const line_circle_radius = 30 #pixel
-const pointerSpeed = 10 #pixel/second
-const pointerOffset = Vector2(0,-30) #pixel
-const skullSpeed = 10 #pixel/second
+const TARGET_ANGLE_EPSILON = 0.1* TAU
```

Comment:
It could be nice to type these const:
const TARGET_ANGLE_EPSILON : float = 0.1* TAU
...